tests/gnrc_tcp_client/README.md

```diff
@@ -16,8 +16,8 @@ The test sequence above runs a configurable amount of times.
 Usage (native)
 ==========
 
-Build and run test:
-make clean all term
+Build and run test (don't forget to use different tap device than server):
+make clean all term PORT=tap1
```

Comment:
such a line should be added to the servers readme, too ... or not?

Reply:
`PORT` defaults to `tap0` so it is not strictly necessary.

Reply:
alternatively, as some might overlook this or do not know that `tap0` is default, we could add:

```
ifeq (native,$(BOARD))
PORT ?= tap1
endif
```

in the Makefile instead